config/jinja2.py

```diff
@@ -205,6 +303,8 @@ def environment(**options):
             "apply_generic_xsl": apply_generic_xsl,
             "parse_json": parse_json,
             "tna_html": tna_html,
+            "remove_string_case_insensitive": remove_string_case_insensitive,
+            "truncate_preserve_mark_tags": truncate_preserve_mark_tags_tags,
```

Comment:
```suggestion
            "truncate_preserve_mark_tags": truncate_preserve_mark_tags,
```

Reply:
Thanks @TNA-Allan 🤦🏽 - I can't entirely blame "replace all" for this, but that's at least partly how this happened. I've pushed up an amended commit.